Cargo.toml

```diff
@@ -13,6 +13,15 @@ panic = "abort"
 [features]
 salustsm = []
 
+[[bin]]
+name = "salus-tsm"
+path = "src/salus_tsm.rs"
+required-features = ["salustsm"]
+
+[[bin]]
+name = "salus"
+path = "src/main.rs"
```

Comment:
Instead of `main.rs` and `salus_tsm.rs`, can we put `main` in both names so that it's obvious that they're both the "main" modules of their respective binaries?